jwst/refpix/irs2_subtract_reference.py

```diff
@@ -74,6 +72,7 @@ def correct_model(input_model, irs2_model,
     # Only copy in SCI data array for now; that's all we need. The rest
```

Comment:
Might want to update this comment, now that both SCI and PIXELDQ are being copied.

Reply:
Done, thanks for pointing that out.